charts/nm-vllm-production-monitoring/examples/full-suite-demo/values.yaml

```diff
@@ -0,0 +1,35 @@
+grafana:
+  adminPassword: password
+  service:
+    type: LoadBalancer
+
+nm-vllm:
+  apiServer:
+    extraArgs:
+      - --dtype
+      - float16
+
+  modelName: TheBloke/Mistral-7B-Instruct-v0.2-GPTQ
+
+  nodeSelector:
+    nvidia.com/gpu.product: NVIDIA-A100-SXM4-40GB
+
+  resources:
+    limits:
+      cpu: 4
+      memory: 16Gi
+      nvidia.com/gpu: 1
+
+prometheus:
+  alertmanager:
+    enabled: false
+  prometheus-node-exporter:
+    enabled: false
+  server:
+    global:
+      evaluation_interval: 30s
+      scrape_interval: 10s
```

Comment:
During exploration, it turned out having values like these was important to getting the dashboard to work righ